src/lib/profiles/data-management/Current/NotificationEngine.h

```diff
@@ -266,8 +266,6 @@ class NotificationEngine
         NotifyRequestBuilderState mState;
         PacketBuffer * mBuf;
         SubscriptionHandler * mSub;
-        uint32_t mMaxNotificationSize;
```

Comment:
These were introduced as a part of the SubscriptionlessNotify changeset; however, it looks like they are not needed because the are computed from the Binding/Packet buffer